server/mvc/models/User/RegionProfileMapper.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+var Model;
+
+var getModel = function(thinky){
+  if (Model === undefined) {
+    Model = thinky.createModel("RegionProfiles", {
+      userProfileId: thinky.type.number().integer().default(null).min(1).required().allowNull(false),
+      userId: thinky.type.number().integer().default(null).min(1).required().allowNull(false),
+      region: thinky.type.string().default(null).min(1).required().allowNull(true).validator(function(homeRegion){
+        return (
+          homeRegion === null
+          || (
+            typeof(homeRegion) == "string"
+            && (["us", "eu"].indexOf(homeRegion) > -1)
+          )
+        );
+      }),
+      characterName: thinky.type.string().default(null).required().allowNull(true).validator(function(nickname){
```

Comment:
Ah, the use of the battletag here was just a filler until we ask the user to type in a Starbow nickname.  Thanks for the regex pattern :D